brave/brave-core
# Fixes exclude being out of sync in ac table

Resolves https://github.com/brave/brave-browser/issues/3510
Resolves https://github.com/brave/brave-browser/issues/3524

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on

## Test Plan:
- clean profile
- enable rewards
- add couple sites to ac table
- remove a site from ac table
- make sure that sum is 100%
- go to the site that you removed, open panel and include it back
- make sure that site is back in ac table and sum is 100%
- make sure that exclude count is always correct

## Reviewer Checklist:
